services/managers/evewho_manager.py

```diff
@@ -0,0 +1,19 @@
+
+from django.conf import settings
+
+import logging
+import requests
+import json
+
+
+class EveWhoManager():
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def get_corporation_members(corpid):
+        url = "http://evewho.com/api.php?type=corplist&id=%s" % corpid
+        jsondata = requests.get(url).content
+        data=json.loads(jsondata.decode())
+
+        return {row["character_id"]:{"name":row["name"], "id":row["character_id"]} for row in data["characters"]}
```

Comment:
are we sure that this will not tax the evewho website to much? should we add some kind of cache so that it only updates every so often?